packages/manager/modules/dbaas-logs/src/logs/detail/home/home.constants.js

```diff
@@ -0,0 +1,17 @@
+export const STATUS = {
+  DISABLED: 'DISABLED',
+  ENABLED: 'ENABLED',
+  INIT: 'INIT',
+  TO_CONFIG: 'TO_CONFIG',
+};
+
+export const STATUS_CLASS = {
+  [STATUS.DISABLED]: 'oui-badge_error',
+  [STATUS.ENABLED]: 'oui-badge_success',
+  [STATUS.INIT]: 'oui-badge_info',
+  [STATUS.TO_CONFIG]: 'oui-background-g-100',
+};
+
+export default {
+  STATUS_CLASS,
+};
```

Comment:
```suggestion
export default {
  STATUS_CLASS,
  STATUS,
};
```

Reply:
the values of STATUS are unused in view or controller. For now it only serves to get the corresponding class